scripts/lint/ourlint.js

```diff
@@ -225,6 +225,60 @@ const TESTS = {
       return `Bad licenses:\n\n${badLicenses.join('\n')}\n\nSee the valid license indentifiers at https://spdx.org/licenses`;
     }
   },
+
+  /* Ensure that all fields are sorted in the same way for all icons */
+  fieldsSorted(_, iconsDataString) {
```

Comment:
You can load the JSON as an array of icons and use the `Object.keys(icon)` to get the fields.

```js
const icons = json.icons;
for (const icon of icons) {
	const keys = Object.keys(icon);
	// ... sort and stringify comparison
}
```

Reply:
Are guaranteed that the fields are sorted in the same order than the file?

Reply:
Okay, I've refactored with your suggestion.

Reply:
Yes. This should not be a problem when using `JSON.parse`. We have been parsing and stringifying data in `add-icon-data` for a long time and have not seen any problems.

The order only changes during the `JSON.stringify` with a number-keys object. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify.